### ISSUE TITLE
Enable reading platform configuration from configuration file in the content repository

### ISSUE THREAD

**Issue by JaroCamphuijsen** [esciencecenter-digital-skills/NEBULA#12]
An example of the content repository configuration file would be in `config.json`:

```
{
    "publicProps": {
        "title": "NEBULA test platform",
        "repoName":"research-software-support",
        "repoOwner": "esciencecenter-digital-skills",
        "repoTag": "v0.1.0",
        "categoryOrder": ["Getting Started", "Reusability", "Publishing & Citing", "Upcoming (under construction)"]
    }
}
```

**Comment by JaroCamphuijsen** [comment]
This is probably also related to and maybe even dependent on #4

**Comment by JaroCamphuijsen** [comment]
This was closed as per #13
